analysis/src/Shared.ml

```diff
@@ -49,12 +49,12 @@ let declToString ?(recStatus = Types.Trec_not) name t =
 let cacheTypeToString = ref false
 let typeTbl = Hashtbl.create 1
 
-let typeToString (t : Types.type_expr) =
+let typeToString ?(lineWidth = 60) (t : Types.type_expr) =
```

Comment:
It's the second time there's `60`.
First: unique source of truth.
Second: do we have slightly redundant printing functions? Why 2?

Reply:
Right, I don't really need the default itself here. Fixing.

Reply:
Done.